src/CoreWCF.Primitives/src/CoreWCF/IdentityModel/Tokens/DefaultTokenReplayCache.cs

```diff
@@ -0,0 +1,86 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using Microsoft.Extensions.Caching.Memory;
+
+namespace CoreWCF.IdentityModel.Tokens
+{
+    /// <summary>
+    /// A default implementation of the Token replay cache that is backed by
+    /// a bounded cache.
+    /// </summary>''''
+    /// 
+    internal class DefaultTokenReplayCache : TokenReplayCache
+    {
+        private static readonly int DefaultTokenReplayCacheCapacity = 500000;
+        private static readonly TimeSpan DefaultTokenReplayCachePurgeInterval = TimeSpan.FromMinutes(1);
+        private MemoryCache _memoryCache;
+
+        /// <summary>
+        /// Constructs the default token replay cache.
+        /// </summary>
+        public DefaultTokenReplayCache()
+            : this(DefaultTokenReplayCache.DefaultTokenReplayCacheCapacity, DefaultTokenReplayCache.DefaultTokenReplayCachePurgeInterval)
+        { }
+
+        /// <summary>
+        /// Constructs the default token replay cache with the specified 
+        /// capacity and purge interval.
+        /// </summary>
+        /// <param name="capacity">The capacity of the token cache</param>
+        /// <param name="purgeInterval">The time interval after which the cache must be purged</param>
+        public DefaultTokenReplayCache(int capacity, TimeSpan purgeInterval)
+            : base()
+        {
+            _memoryCache = new MemoryCache(new MemoryCacheOptions
+            {
+                SizeLimit = DefaultTokenReplayCacheCapacity,
+                ExpirationScanFrequency = DefaultTokenReplayCachePurgeInterval
```

Comment:
You aren't using the passed in capacity and purgeInterval values when constructing the MemoryCacheOptions

Reply:
You still haven't fixed this